include/vfio-user.h

```diff
@@ -163,33 +163,33 @@ struct vfio_user_irq_info {
 #define VFIO_REGION_SUBTYPE_MIGRATION           (1)
 
 struct vfio_device_migration_info {
-	__u32 device_state;         /* VFIO device state */
-	__u32 reserved;
-	__u64 pending_bytes;
-	__u64 data_offset;
-	__u64 data_size;
+	uint32_t device_state;         /* VFIO device state */
```

Comment:
I left these with __ on purpose, as they're from the kernel headers.

Reply:
Then maybe we should use them from linux/vfio.h?